package.json

```diff
@@ -203,12 +203,6 @@
     "webpack-hot-middleware": "^2.18.2",
     "xml2json": "^0.11.0"
   },
-  "pre-commit": [
-    "lint-staged"
-  ],
-  "pre-push": [
-    "validate"
```

Comment:
I agree with this change... CI should do this after push, but _before_ merging. This will also make invalid code much more visible, since it'll be a badge at the bottom of the PR on the PR screen for everyone to see.